osu.Game/Screens/OnlinePlay/Multiplayer/Match/ReadyButton.cs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using JetBrains.Annotations;
+using osu.Framework.Allocation;
+using osu.Framework.Localisation;
+using osu.Framework.Threading;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Backgrounds;
+using osu.Game.Online.Multiplayer;
+
+namespace osu.Game.Screens.OnlinePlay.Multiplayer.Match
+{
+    public class ReadyButton : Components.ReadyButton
```

Comment:
I tried to rename this to `MultiplayerReadyButton` only to find out... there is also a `MultiplayerReadyButton`.

I see this as a blocker, need to decide on better naming here.

Reply:
Looks like rider was reporting a conflict with a reflected dll... so just renaming to `MultiplayerReadyButton` should be fine.

Reply:
Renamed.